src/main/scala/edu/berkeley/ce/rockslicing/Block.scala

```diff
@@ -212,7 +212,7 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
               A_matx(1, ::) := n2.t
               A_matx(2, ::) := n3.t
               val p_vect = A_matx \ b_vect
-              Some((p_vect(0), p_vect(1), p_vect(2)))
+              Some((p_vect(0) + centerX, p_vect(1) + centerY, p_vect(2) + centerZ))
```

Comment:
Could you explain why you made this change? I just want to be sure I understand what's going on.